src/style.css

```
@@ -1 +1,31 @@
 /* write your styles here */
+body {
+  margin: 0;
+}
+
+body > * {
+  box-sizing: border-box;
+  width: 300px;
+  height: 100px;
+  margin: auto;
```

Вертикальні відступи необов'язкові
```suggestion
  margin: 0 auto;
```

---

сенкс)))